### DISCUSSION TITLE
A problem with “stochastic”

### DISCUSSION THREAD

**Discussion by bendibendi** [idaholab/moose#23812] [Q&A Modules: General] [answered]
hello, I'm a beginner in MOOSE and I think I'm having a problem with the official routines. Recently I was trying to run the official multi-physics field coupling routine stochastic and this is its URL：https://mooseframework.inl.gov/modules/combined/examples/stm_thermomechanics.html.

I ran the input file "graphite_ring_thermomechanics.i" in the "stochastic" directory using the file "combined-opt" in the "combined" directory:
<img width="345" alt="image" src="https://user-images.githubusercontent.com/115389604/226888215-c0263f6d-9ff2-4d70-8e27-8cc36dfae78c.png">
This is my compile command line：
<img width="650" alt="image" src="https://user-images.githubusercontent.com/115389604/226890518-2e27d98a-1ced-4897-8fad-a13a06bb6038.png">
I ended up with an output file, and I used paraview to look at this mesh file and found that the results were not the same as the official routines.
Here are the official results.
<img width="325" alt="image" src="https://user-images.githubusercontent.com/115389604/226893530-e33347a1-584f-49ab-9b90-8c8d4d594ea6.png">
<img width="298" alt="image" src="https://user-images.githubusercontent.com/115389604/226893716-b800bf26-b5ef-4fba-a929-bec95bc50bec.png">
Here are my results.
<img width="294" alt="image" src="https://user-images.githubusercontent.com/115389604/226893893-4ee8f396-2e65-4c11-bb75-b01785eecf05.png">
<img width="289" alt="image" src="https://user-images.githubusercontent.com/115389604/226894259-20879595-5aa2-4c57-8477-0409d046347c.png">

The results of physical quantities such as T and dispxd are different from the official routines.I wonder  what is wrong with this?

**Comment by GiudGiud** [comment, marked as answer]
Hello

paraview opens the first time step, initialization, by default. Did you navigate to the final time step?

Guillaume

**Reply by bendibendi** [reply]
Thank you a lot. I am not proficient enough with paraview. I just navigated through the last time step and observed the correct results.
